gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaSource.java

```diff
@@ -389,10 +466,21 @@ private void startAtEarliestOffset() {
     private void startAtLatestOffset() {
       this.startOffset = this.latestOffset;
     }
+  }
 
-    private long getStartOffset() {
-      return this.startOffset;
+  /**
+   * Implementation of {@link Predicate} that takes in a Kafka topic name via its constructor. It returns true in the
+   * {@link Predicate#apply(Object)} method only if the given {@link KafkaTopic} matches the specified topic name.
+   */
+  @AllArgsConstructor
+  private static class KafkaTopicNamePredicate implements Predicate<KafkaTopic> {
+
+    @NonNull
+    private final String topicName;
+
+    @Override
+    public boolean apply(KafkaTopic input) {
+      return input.getName().equals(this.topicName);
```

Comment:
please make it support regex so that we can specify `Voyager.*` instead of each individual topic.